code/API_definitions/quality-on-demand.yaml

```diff
@@ -197,24 +225,6 @@ paths:
                     status: 400
                     code: INVALID_ARGUMENT
                     message: "Expected property is missing: device"
-                InsufficientDeviceProperties:
```

Comment:
We should deprecate as well DeviceMissing, which is covered with better dedicated errors

Reply:
Agreed, done as part of https://github.com/camaraproject/QualityOnDemand/pull/326/commits/061776de7e60d68b91967e815413adc88e37651b